common/browser_utilities.py

```diff
@@ -0,0 +1,10 @@
+def is_text_present_with_retry(browser, retries: int, text: str, delay: int) -> bool:
```

Comment:
Don't need to pass in browser it's global, also I think it's worth defaulting the retries and the delay

Reply:
Also we could take a lambda rather than just checking for text as this could hide bugs if there is the same text multiple times on the page?